devtools-ci-index.yaml

```diff
@@ -1789,6 +1837,24 @@
             branches:
                 - origin/master
     <<: *che-functional-tests-template
+    
+
+- job-template:
+    name: '{ci_project}-{git_repo}-periodic-{env}-{cluster}'
+    triggers:
+        - timed: 'H 1,13 * * *'
+    publishers:
```

Comment:
Not sure if email publishers work on CICO. Not a blocker. Lets try this out though